Gemfile.lock

```diff
@@ -1058,6 +1058,7 @@ DEPENDENCIES
   sidekiq-pro!
   sidekiq-scheduler (~> 3.0)
   simplecov (< 0.18)
+  slack-notify
```

Comment:
Did you mean to commit this?

Reply:
No, not sure how that ended up in there.